java/src/jmri/jmrix/can/cbus/CbusSensor.java

```diff
@@ -82,40 +82,32 @@ public void requestUpdateFromLayout() {
     }
 
     /**
-     * User request to set the state, which means that we broadcast that to all
-     * listeners by putting it out on CBUS. In turn, the code in this class
+     * User request to set the state.
+     * We broadcast that to all listeners by putting it out on CBUS. 
+     * In turn, the code in this class
      * should use setOwnState to handle internal sets and bean notifies.
-     * Unknown state does not send a message to CBUS but updates 
-     * internal sensor state, enabling user test of Start of Day / Logix.
+     * Unknown / intermittent states do not send a message to CBUS,
```

Comment:
should this be "inconsistent" instead of "intermittent"?

Reply:
Thanks for spotting, have updated.